TrafficCapture/captureOffloader/src/main/java/org/opensearch/migrations/trafficcapture/StreamChannelConnectionCaptureSerializer.java

```diff
@@ -190,40 +190,19 @@ public CompletableFuture<Object> flushCommitAndResetStream(boolean isFinal) thro
         return future;
     }
 
-    /**
-     * This call is BLOCKING.  Override the Closeable interface - not addCloseEvent.
-     *
-     * @throws IOException
-     */
-    @Override
-    public void close() throws IOException {
```

Comment:
Surprised we don't call this anywhere. I guess we close the channel in places but never the Serializer...